### DISCUSSION TITLE
System requirements

### DISCUSSION THREAD

**Discussion by jon4hz** [mbround18/valheim-docker#245] [Q&A] [answered]
Hi,

What are the minimum requirements for running the server? Is there a guideline for resources per user?

Thanks in advance :)

**Comment by mbround18** [comment]
Use discussions, transferring this to discussions.

**Comment by CosmicHorrorDev** [comment, marked as answer]
What all specifically are you looking at? Are you planning on running on a  like with AWS, GCP, DO droplet, etc. or are you trying to judge if hardware you have will do the trick?

**Reply by jon4hz** [reply]
I want to install the server on a VPS. Since I can't forward any ports on my router it's the best option imo.

**Reply by mbround18** [reply]
Recommended is 1-2 cores with 2-4 gb of ram. 1-5 players for ~2gb ram and ~4gb  for 10 players

if you plan on using mods, bank on 4gb of ram. My servers sit idle at 1-2gb of ram but go up with more connections.
